Src/coffeescript/cql-execution/package.json

```diff
@@ -18,7 +18,8 @@
   "devDependencies": {
     "coffee-script": "^1.8.0",
     "mocha": "^1.21.5",
-    "should": "^4.0.4"
+    "should": "^4.0.4",
+    "coveralls": "^2.11.2",
```

Comment:
I have no idea how that got in there.  The only thing I actually touched in that file was updating the version number.